libs/javascript.json

```diff
@@ -2,14 +2,16 @@
   {"id":"js.async","imports":["async"],"name":"async","repo":"caolan/async","tags":["async"],"tech":["app-arch"],"status":"model-ready"},
   {"id":"js.graphql","imports":["graphql"],"name":"graphql","repo":"graphql/graphql-js","tags":["query-language"],"tech":["app-arch"],"status":"model-ready"},
   {"id":"js.react-native","imports":["react-native"],"name":"react-native","repo":"facebook/react-native","tags":["framework"],"tech":["app-arch"],"status":"model-ready"},
-  {"id":"js.webpack","imports":[],"name":"webpack","repo":"webpack/webpack","tags":["module-bundler"],"tech":["software-distribution"],"status":"awaiting-model"},
   {"id":"js.pdf","imports":["pdf"],"name":"pdf","repo":"mozilla/pdf.js","tags":["pdf-reader"],"tech":["app-blocks"],"status":"model-ready"},
   {"id":"js.normalizr","imports":["normalizr"],"name":"normalizr","repo":"paularmstrong/normalizr","tags":["transformations"],"tech":["app-data"],"status":"model-ready"},
   {"id":"js.yaml","imports":["yaml"],"name":"yaml","repo":"jeremyfa/yaml.js","tags":["json-yaml"],"tech":["app-data"],"status":"model-ready"},
+  {"id":"js.chalk","imports":["chalk"],"name":"Chalk","repo":"chalk/chalk","tags":["terminal-styling"],"tech":["cli"],"status":"awaiting-model"},
+  {"id":"js.commander","imports":["commander"],"name":"Commander.js","repo":"tj/commander.js","tags":["node-program"],"tech":["cli"],"status":"awaiting-model"},
   {"id":"js.tesseract","imports":["tesseract"],"name":"tesseract","repo":"naptha/tesseract.js","tags":["ocr"],"tech":["computer-vision"],"status":"model-ready"},
   {"id":"js.d3","imports":["d3"],"name":"d3","repo":"d3/d3","tags":["visualization"],"tech":["data-science"],"status":"model-ready"},
   {"id":"js.rickshaw","imports":["rickshaw"],"name":"rickshaw","repo":"shutterstock/rickshaw","tags":["visualization"],"tech":["data-science"],"status":"model-ready"},
   {"id":"js.winston","imports":["winston"],"name":"winston","repo":"winstonjs/winston","tags":["logging"],"tech":["diagnostics"],"status":"model-ready"},
+  {"id":"js.eslint","imports":["eslint"],"name":"Eslint","repo":"eslint/eslint","tags":["eslint"],"tech":["eng-tools"],"status":"awaiting-model"},
```

Comment:
this does not seem to be a library

Reply:
it has nodejs API https://eslint.org/docs/developer-guide/nodejs-api#nodejs-api, so could be used as a library